src/scss/elements/base.scss

```diff
@@ -4,4 +4,5 @@ body {
     margin: 0;
```

Comment:
Úplne nahoře máš zbytečný prázdný řádek. Vím, už jsem s tím otravný, ale prostě to nevypadá dobře. :)

Reply:
Přidej si sem i další užitečný base elements nastavení. U mě to třeba na jedno projektu vypadá takhle...

body {
    margin: 0;
    font-size: 16px;
    scroll-behavior: smooth;
    width: 100%;
    height: 100%;
    font-family: 'Hanken Grotesk', sans-serif;
    overflow-x: clip;
    box-sizing: border-box;
    scroll-padding-top: 70px;
    background: $gray-1;
}

h1, h2, h3, h4, h5, h6 {
    padding: 0;
    margin: 0;
}
  
p {
    padding: 0;
    margin: 0;
}

Reply:
Smazala jsem řádek a upravila base elements